### ISSUE TITLE
Android 33 issue when initializing 

### ISSUE THREAD

**Issue by strafton** [apache/cordova-plugin-geolocation#264]
# Bug Report
I am seeing errors my cordova app with Android 33 API. It appears to never ask for the locations services permissions so there is no way to set these on the device even manually.  This has worked on all other Android API's for multiple YEARS without issue.

Any thoughts on getting this to work would be appreciated!

## Problem
Location services is declined always and the user is never presented a popup to accept these due to the an issue with the thread pool.

### What is expected to happen?
The app initialized and asks to allow locations services.

### What does actually happen?
The popup for location services is never presented and the permissions is denied.

## Information

D/GeolocationPlugin: We are entering execute
W/PluginManager: THREAD WARNING: exec() call to Geolocation.getPermission blocked the main thread for 30ms. Plugin should use CordovaInterface.getThreadPool().
D/CordovaActivity: Paused the activity.
D/GeolocationPlugin: Permission Denied!

### Environment, Platform, Device

Android 33+  emulators and physical devices

### Version information

Android Studio Electric Eel | 2022.1.1 Patch 2

 cordova -v
11.1.0

@havesource/cordova-plugin-push 4.0.0-dev.0 "Cordova Push Plugin"
cordova-custom-config 5.1.1 "cordova-custom-config"
cordova-launch-review 4.0.1 "Launch Review"
cordova-plugin-ble-central 1.6.3 "BLE"
cordova-plugin-device 2.1.0 "Device"
cordova-plugin-geolocation 4.1.0 "Geolocation"
cordova-plugin-inappbrowser 5.0.0 "InAppBrowser"
cordova-plugin-keyboard 1.2.0 "Keyboard"
cordova-plugin-splashscreen 6.0.2 "Splashscreen"
cordova-plugin-statusbar 3.0.0 "StatusBar"

**Comment by breautek** [comment]
I think this might be related to: https://github.com/apache/cordova-plugin-geolocation/pull/250

Could you give the dev version of this plugin a try? That PR is already merged into master, just a release hasn't been made yet.

```
cordova plugin remove cordova-plugin-geolocation
cordova plugin add https://github.com/apache/cordova-plugin-geolocation.git
```

**Comment by strafton** [comment]
Thank you. I ended up using the dev version for a production build which is not idea but it was the only solution.Sent from my iPhoneOn Sep 14, 2023, at 8:32 AM, Norman Breau ***@***.***> wrote:﻿
I think this might be related to: #250
Could you give the dev version of this plugin a try? That PR is already merged into master, just a release hasn't been made yet.
cordova plugin remove cordova-plugin-geolocation
cordova plugin add https://github.com/apache/cordova-plugin-geolocation.git

**Comment by breautek** [comment]
fixed by #250
